.github/workflows/terraform.yml

```diff
@@ -0,0 +1,44 @@
+# This workflow installs the latest version of Terraform CLI. On pull request events, this workflow will run
+# `terraform init`, `terraform fmt`, and `terraform plan`.
+#
+# Documentation for `hashicorp/setup-terraform` is located here: https://github.com/hashicorp/setup-terraform
+
+name: 'Terraform'
+
+on:
+  push:
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  terraform:
+    name: 'Terraform'
+    runs-on: ubuntu-latest
```

Comment:
I thought we were versioning, so that we would have stability?

Reply:
That's an interesting question. For anything we build (or consume as part of our apps), it often seems helpful to specify a major version. For the host OS running our tests... good question.

For now I'll leave it as-is, because this is the working example I received from another terraform module we've published, and it doesn't seem worth the time to learn how to specify a different version right now. Thanks, though.

Reply:
I updated it on this (and other repos) to `ubuntu-22.04` which is the current for `latest`.